src/main/java/com/sngular/kloadgen/property/editor/ValueSerializerPropertyEditor.java

```diff
@@ -63,8 +62,11 @@ private void fillSerializer(final JComboBox<String> objectJComboBox) {
         new ConfigurationBuilder()
             .addUrls(ClasspathHelper.forClass(Serializer.class))
             .filterInputsBy(new FilterBuilder()
-                                .includePackage("com.sngular.kloadgen.serializer")
-                                .includePackage("io.confluent.kafka.serializers"))
+                                .includePackage(SerDesPackageValue.COM_SNGULAR_KLOADGEN_SERIALIZER)
+                                .includePackage(SerDesPackageValue.IO_CONFLUENT_KAFKA_SERIALIZERS)
+                                .includePackage(SerDesPackageValue.IO_APICURIO_REGISTRY_SERDE_AVRO_AVRO_KAFKA_DESERIALIZER)
+                                .includePackage(SerDesPackageValue.IO_APICURIO_REGISTRY_SERDE_JSONSCHEMA_JSON_SCHEMA_KAFKA_DESERIALIZER)
+                                .includePackage(SerDesPackageValue.IO_APICURIO_REGISTRY_SERDE_PROTOBUF_PROTOBUF_KAFKA_DESERIALIZER))
```

Comment:
Should these be `DESERIALIZER` on the serializer? If I recall correcly they were different before this last commit